src/hotspot/share/opto/vectornode.hpp

```diff
@@ -561,6 +593,14 @@ class MinVNode : public VectorNode {
   virtual int Opcode() const;
 };
 
+class UMinVNode : public VectorNode {
+ public:
+  UMinVNode(Node* in1, Node* in2, const TypeVect* vt) : VectorNode(in1,in2,vt) {
```

Comment:
```suggestion
  UMinVNode(Node* in1, Node* in2, const TypeVect* vt) : VectorNode(in1, in2, vt) {
```
spaces required by style guide